addons/mrp_account/__manifest__.py

```diff
@@ -22,12 +22,12 @@
     "data": [
         'security/ir.model.access.csv',
         "views/product_views.xml",
-        "views/mrp_production_views.xml",
         "views/analytic_account_views.xml",
         "views/account_move_views.xml",
         "views/mrp_workcenter_views.xml",
         "report/report_mrp_templates.xml",
         "wizard/mrp_wip_accounting.xml",
+        "views/mrp_production_views.xml",
```

Comment:
I wouldn't do that in stable. This enters the domain of improvement so -> master

Reply:
It doesn't feel right but TIC told me there's no better way to do it. (And yeah, it was supposed to be a bugfix PR but implementation subtasks were added to the task, so the title is misleading now...)

Reply:
I don't understand, do you _need_ to move it there? Is it related to `"wizard/mrp_wip_accounting.xml",` just above?